depthai_nodes/ml/parsers/yolo.py

```diff
@@ -107,10 +187,39 @@ def setNumKeypoints(self, n_keypoints):
         """
         self.n_keypoints = n_keypoints
 
+    def setAnchors(self, anchors):
+        """Sets the anchors for the YOLO model.
+
+        @param anchors: The anchors for the YOLO model.
+        @type anchors: List[np.ndarray]
+        """
+        self.anchors = anchors
+
+    def setYoloVersion(self, yolo_version):
```

Comment:
Add types assertions and returns like setOutputLayerNames

Reply:
done